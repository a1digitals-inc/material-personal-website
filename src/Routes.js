import React from "react";
import { Switch, Route } from "react-router-dom";

import Home from "./pages/Home";
import Portfolio from "./pages/Portfolio";
import Blog from "./pages/Blog";

const Routes = (props) => {
  return (
    <Switch>
      <Route path="/blog">
        <Blog {...props} />
      </Route>
      <Route path="/portfolio">
        <Portfolio {...props} />
      </Route>
      <Route path="/">
        <Home {...props} />
      </Route>
    </Switch>
  );
};

export default Routes;
