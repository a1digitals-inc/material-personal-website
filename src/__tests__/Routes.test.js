import React from "react";
import { shallow } from "enzyme";

import Routes from "../Routes";

it("renders without crashing", () => {
  const wrapper = shallow(<Routes />);
  expect(wrapper).toMatchSnapshot();
});
