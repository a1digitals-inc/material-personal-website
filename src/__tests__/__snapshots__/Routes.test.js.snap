// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component should render without crashing 1`] = `
<Switch>
  <Route
    path="/portfolio"
  >
    <Portfolio />
  </Route>
  <Route
    path="/"
  >
    <Home />
  </Route>
</Switch>
`;
